packages/xstate-wallet/src/messaging.ts

```diff
@@ -85,6 +87,11 @@ export class MessagingService implements MessagingServiceInterface {
     this.eventEmitter.emit('SendMessage', response);
   }
 
+  public async sendError(id: number, error: ErrorResponse['error']) {
+    const response = {id, jsonrpc: '2.0', error} as ErrorResponse; // typescript can't handle this otherwise
```

Comment:
Any idea why?

Reply:
It's probably the `'2.0'` being typed as `string` in `response`.
```suggestion
    const response: Response = {id, jsonrpc: '2.0', error};
```

If you assign the type `Response` to `response`, it has a similar effect while providing type safety -- `'1.0'` is _not_ assignable to `'2.0'`, and the types of the other properties must match as well.